### ISSUE TITLE
reported AUC

### ISSUE THREAD

**Issue by elsobhano** [VISION-SJTU/RECCE#24]
Hi 
I have a question about Your results in tabel 4.
when you are reporting AUC, does your test datsets contain real images?
Becaus in terms of AUC we need to positive and negative samples.

**Comment by XJay18** [comment]
Hi, 
Thanks for your interest.

In Table 4 of the manuscript, we evaluate the model with real images and the specific forged images from the test set.
For example, "Train (DF) Test (F2F)" means that we train the model with real images and DF images from the training set and test the model with real images and F2F images from the test set.

**Comment by XJay18** [comment]
Close due to inactivation. Please feel free to reopen this issue if you still have related problems.
